### PR TITLE
Added a basic sample for the amp-access component.

### PR DESCRIPTION
**Pull request by andreban**

### CODE REVIEW

File: src/20_Components/amp-access.html [hunk 1]
```diff
@@ -0,0 +1,81 @@
...
+  
+  <!-- #### Setup -->
+  <!-- Import the `amp-access` component in the header. -->
+  <script async custom-element="amp-access" src="https://cdn.ampproject.org/v0/amp-access-0.1.js"></script>  
+
+  <!-- Optionally, use the `amp-mustache` component to show text with values replaced from
+  the authorization response -->
+  <script async custom-template="amp-mustache" src="https://cdn.ampproject.org/v0/amp-mustache-0.1.js"></script>
+
+  <!-- Add the configuration for the `authorization`, `pingback` and the `login` endpoints. It is
```
Comment (sebastianbenz) [inline src/20_Components/amp-access.html]: can you add a link to the source code on github?

File: src/20_Components/amp-access.html [hunk 2]
```diff
@@ -0,0 +1,81 @@
...
+    <h3>Article 1</h3>    
+    <p>
+      This text is part of the article preview and visible to all users. 
+      This text is part of the article preview and visible to all users. 
+      This text is part of the article preview and visible to all users. 
+    </p>
+  </div>
+
+   <!-- Use the `amp-access` attribute to control the visibility of each component. The expression
+   must evaluate to a boolean value. If it evaluates to `TRUE`, the section will be show.  -->
```
Comment (sebastianbenz) [inline src/20_Components/amp-access.html]: s/be show/be shown/

Comment (andreban) [inline src/20_Components/amp-access.html]: Fixed

File: src/20_Components/amp-access.html (hunk 1 above)
Comment (andreban) [inline src/20_Components/amp-access.html]: Fixed.

**Comment by andreban** [comment]
PTAL

File: src/20_Components/amp-access.html [hunk 3]
```diff
@@ -0,0 +1,81 @@
...
+
+  <!-- It is possible to combine two values on the evaluation expression.
+    Check the docs to find out more about the [Expression Grammar](https://github.com/ampproject/amphtml/blob/master/extensions/amp-access/amp-access.md#appendix-a-amp-access-expression-grammar)
+   -->
+  <div amp-access="access AND NOT login">
+    <p>This section is only visible to users with <pre>"access": true</pre> and <pre>"login": false</pre></p>
+  </div>  
+
+  <!-- Combine `amp-access` and `amp-mustache` to show customise the messages with
+  information returned on the authorization request. -->
```
Comment (sebastianbenz) [inline src/20_Components/amp-access.html]: sug: "Combine `amp-access` and `amp-mustache` to show information returned from the authorization request."

File: src/20_Components/amp-access.html [hunk 4]
```diff
@@ -0,0 +1,81 @@
+<!-- #### Introduction
+  The [amp-access](https://github.com/ampproject/amphtml/blob/master/extensions/amp-access/amp-access.md) component allows to control which content is accessible by the reader and with what restrictions.
```
Comment (sebastianbenz) [inline src/20_Components/amp-access.html]: Should we mention paywalls here?

**Comment by sebastianbenz** [comment]
Does it make sense to add a login button?

**Comment by sebastianbenz** [comment]
Added some small suggestions.

**Comment by andreban** [comment]
Implemented suggestions. PTAL.

File: src/20_Components/amp-access.html [hunk 5]
```diff
@@ -0,0 +1,88 @@
+<!-- #### Introduction
+  The [amp-access](https://github.com/ampproject/amphtml/blob/master/extensions/amp-access/amp-access.md) adds support for paywalls and subscriptions to AMP, allowing the publisher to control which content is accessible by the reader and with what restrictions.
```
Comment (sebastianbenz) [inline src/20_Components/amp-access.html]: add component in "The [amp-access](https://github.com/ampproject/amphtml/blob/master/extensions/amp-access/amp-access.md) component adds"

File: src/20_Components/amp-access.html [hunk 6]
```diff
@@ -0,0 +1,88 @@
+<!-- #### Introduction
+  The [amp-access](https://github.com/ampproject/amphtml/blob/master/extensions/amp-access/amp-access.md) adds support for paywalls and subscriptions to AMP, allowing the publisher to control which content is accessible by the reader and with what restrictions. 
+
+  For a sample implementation on the server-side implementation of the amp-access component, check the samples on the [amp-publisher-sample](https://github.com/ampproject/amp-publisher-sample) on Github.
```
Comment (sebastianbenz) [inline src/20_Components/amp-access.html]: s/check/see/g

**Comment by sebastianbenz** [comment]
LGMT

**Pull request merged**